src/Resque/Helpers/CatchOutput.php

```diff
@@ -27,8 +29,8 @@ class CatchOutput extends \Symfony\Component\Console\Output\Output
     /**
      * Constructor.
      *
-     * @param integer                       $verbosity The verbosity level (one of the VERBOSITY constants in OutputInterface)
-     * @param Boolean                       $decorated Whether to decorate messages
+     * @param int                       $verbosity The verbosity level (one of the VERBOSITY constants in OutputInterface)
```

Comment:
Could you please change the alignment of the variables to start with one space after the longest type definition? Thank you 😄

Reply:
done